client-src/elements/chromedash-guide-stage-page.js

```diff
@@ -292,7 +295,7 @@ export class ChromedashGuideStagePage extends LitElement {
     const formattedFeature = formatFeatureForEdit(this.feature);
     return html`
       <form name="feature_form" method="POST"
-        action="/guide/stage/${this.featureId}/${this.stageId}">
+        action="/guide/stage/${this.featureId}/${this.stageId}/${this.intentStage}">
```

Comment:
Is the server-side of this safe for users who already have a tab open on our app?  I can't put my finger on the right line of server-side code, but I want to be sure that someone who had a tab open can still submit.  It used to have the intentStage (called stagId) directly after the feature ID, and I'm not seeing how we are still backwards compatible with that.

Reply:
Sorry, I did not know that was a requirement, but it is possible.

I originally planned to only pass the stage ID in the URL, but realized we also need the intent stage with this new implementation, so I re-added it to the end of the URL, which is why we have it ordered this way. I've made the change in [this commit](https://github.com/GoogleChrome/chromium-dashboard/pull/2561/commits/649d53b54dbfd661191318181ff5cd61d096faff) to the URL address to be specified as `/guide/stage/{featureId}/{intentStage}/{stageId}`, and then to handle the backwards-compatible case where a user submits without a stage ID.